students/julienjass/Lesson07/html_render.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+class Element:
+    """Base Class of the HTML Render"""
+    tag = 'html'
+    ind = 0
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.content = []
+        if content != None:
+            self.content = [content]
+        if kwargs:
+            self.attrs = kwargs
+        else:
+            self.attrs = None
```

Comment:
This could be simplified to this (and would be set to empty dict if there were no kwargs passed in):
```
self.attrs = kwargs
```